studio/components/interfaces/Home/ProjectUsageSection.tsx

```diff
@@ -42,7 +42,7 @@ const ProjectUsageSection = observer(() => {
           <IconLoader className="animate-spin" size={14} />
           <p className="text-sm">Retrieving project usage statistics</p>
         </div>
-      ) : true ? (
+      ) : hasProjectData ? (
```

Comment:
sure?

Reply:
yeap i double checked with Ziinc - `true` was a bug
would imply that new projects wont see the "Welcome" state